### ISSUE TITLE
Telemetry doc metadata.versions.app is `unknown` for all telemetry docs for moh_mali_chw

### ISSUE THREAD

**Issue by kennsippell** [medic/cht-core#8161]
**To Reproduce**
```
SELECT *
FROM couchdb_users_meta
WHERE doc ->> 'type' = 'telemetry' AND doc #>> '{metadata,versions,app}' != 'unknown'
```

There are no results for the disc-mali.ml instance

**Expected behavior**
Version should be known

**Environment**
- Instance: disc-mali.ml
- Version: 4.1

**Comment by kennsippell** [comment]
This is set from `medic/_design/medic-client` doc `deploy_info.version` [here](https://github.com/medic/cht-core/blob/master/webapp/src/ts/services/telemetry.service.ts#L127)

Observing ddoc for the instance there is a `deploy_info` but there is no `version`.

**Comment by dianabarsan** [comment]
I want to assess the urgency of this. I'm about to release 4.2.2 with a critical bug fix. Should this make the cut?

**Comment by Marina-Kenf** [comment]
This is the same for `moh_niger_chw` which is also on 4.x.

**Comment by dianabarsan** [comment]
I am aware. I added all the affects labels.

**Comment by dianabarsan** [comment]
Merged to `master` and backported to 4.2.x
